packages/wrangler/src/dev.ts

```diff
@@ -377,7 +377,7 @@ export const dev = createCommand({
 			{
 				FILE_BASED_REGISTRY: args.experimentalRegistry,
 				MULTIWORKER: Array.isArray(args.config),
-				RESOURCES_PROVISION: false,
+				RESOURCES_PROVISION: args.experimentalProvision ?? false,
```

Comment:
`dev` should ever be provisioning, I don't think?

Reply:
The flag is also used to gate the config id check so it’s needed for now at least otherwise it’ll throw a missing id error